GNUmakefile

```diff
@@ -143,14 +144,22 @@ dev-tree:
 	@$(SHELL) $(CURDIR)/build-support/scripts/dev.sh $(DEV_PUSH_ARG)
 
 cov:
-	go test ./... -coverprofile=coverage.out
+	go test $(GOMODULES) -coverprofile=coverage.out
 	go tool cover -html=coverage.out
 
 test: other-consul dev-build vet test-install-deps test-internal
 
 test-install-deps:
 	go test -tags '$(GOTAGS)' -i $(GOTEST_PKGS)
 
+update-vendor:
+	@echo "--> Running go mod vendor"
+	@go mod vendor
+	@echo "--> Removing vendoring of our own nested modules"
+	@rm -rf vendor/github.com/hashicorp/consul
+	@grep -v "hashicorp/consul/" < vendor/modules.txt > vendor/modules.txt.new
```

Comment:
In one line:
```
sed -i '/github.com\/hashicorp\/consul\//d' vendor/modules.txt 
```

Reply:
But `sed -i` operates differently on linux vs bsd/macos so when there is an easy way to do the same thing without it I take that route.

Reply:
There is a `sed_i` function in our build scripts to abstract away the platform differences.

Reply:
This is why we can't have nice things.